domains/snnttechv.is-local.org.json

```diff
@@ -0,0 +1,13 @@
+{
+    "domain": "snnttechv.is-local.org",
```

Comment:
This should just be the domain you are using, not the whole subdomain.

Reply:
domain is right? or wrong? please tell me, SiR.

Reply:
The domain is right, you just need `is-local.org` only there.